src/openfluid/ui/waresdev/WareSrcWidgetCollection.hpp

```diff
@@ -127,13 +127,20 @@ class MigrationWorker : public QObject
         if (m_CheckoutNew)
         {
           GitUIProxy Git;
-          if (Git.checkout(m_WarePath, GitUIProxy::getCurrentOpenFLUIDBranchName(), true))
+          if (Git.checkout(m_WarePath, GitUIProxy::getCurrentOpenFLUIDBranchName(), false))
           {
             emit info(tr("Successful checkout of the current OpenFLUID version branch"));
           }
           else
           {
-            emit error(tr("Unable to checkout branch corresponding to current OpenFLUID version branch."));
+            if (Git.checkout(m_WarePath, GitUIProxy::getCurrentOpenFLUIDBranchName(), true))
+            {
+              emit info(tr("Successful checkout of the current OpenFLUID version branch"));
```

Comment:
factorize info emission, eg use a status bool and check it once after operations